code/datums/supplypacks.dm

```diff
@@ -326,20 +326,65 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 /////// Weapons: Specialist
 
 /datum/supply_packs/security/armory/ballistic
+	name = "Riot Shotguns Crate"
+	contains = list(/obj/item/weapon/gun/projectile/shotgun/riot,
+					/obj/item/weapon/gun/projectile/shotgun/riot,
+					/obj/item/weapon/gun/projectile/shotgun/riot,
```

Comment:
I'm a bit skeptical of this, to be honest; the main drawback of shotguns is that not every single officer can carry one around without consequene--only two officers.

Making them commonly available not only removes that drawback, but the pricepoint of these is going to encourage players to go with this over the combat.